test/SkipCallsTestHook.t.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {PoolId, PoolIdLibrary} from "../src/types/PoolId.sol";
+import {Hooks} from "../src/libraries/Hooks.sol";
+import {FeeLibrary} from "../src/libraries/FeeLibrary.sol";
+import {IPoolManager} from "../src/interfaces/IPoolManager.sol";
+import {IFees} from "../src/interfaces/IFees.sol";
+import {IHooks} from "../src/interfaces/IHooks.sol";
+import {PoolKey} from "../src/types/PoolKey.sol";
+import {PoolManager} from "../src/PoolManager.sol";
+import {PoolSwapTest} from "../src/test/PoolSwapTest.sol";
+import {Deployers} from "./utils/Deployers.sol";
+import {IDynamicFeeManager} from "././../src/interfaces/IDynamicFeeManager.sol";
+import {GasSnapshot} from "forge-gas-snapshot/GasSnapshot.sol";
+import {DynamicFeesTestHook} from "../src/test/DynamicFeesTestHook.sol";
+import {Currency, CurrencyLibrary} from "../src/types/Currency.sol";
+import {MockERC20} from "solmate/test/utils/mocks/MockERC20.sol";
+import {Constants} from "../test/utils/Constants.sol";
+import {SkipCallsTestHook} from "../src/test/SkipCallsTestHook.sol";
+
+contract SkipCallsTest is Test, Deployers, GasSnapshot {
+    using PoolIdLibrary for PoolKey;
+
+    /// 1111 1111 1100
+    SkipCallsTestHook skipCallsTestHook = SkipCallsTestHook(
+        address(
+            uint160(0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF)
+                & uint160(
+                    ~Hooks.BEFORE_INITIALIZE_FLAG & ~Hooks.AFTER_INITIALIZE_FLAG & ~Hooks.BEFORE_ADD_LIQUIDITY_FLAG
+                        & ~Hooks.AFTER_ADD_LIQUIDITY_FLAG & ~Hooks.BEFORE_REMOVE_LIQUIDITY_FLAG
+                        & ~Hooks.AFTER_REMOVE_LIQUIDITY_FLAG & ~Hooks.AFTER_SWAP_FLAG & ~Hooks.BEFORE_DONATE_FLAG
+                        & ~Hooks.AFTER_DONATE_FLAG
+                )
+        )
+    );
+
+    function setUp() public {
+        SkipCallsTestHook impl = new SkipCallsTestHook();
+        vm.etch(address(skipCallsTestHook), address(impl).code);
+        deployFreshManagerAndRouters();
+        skipCallsTestHook.setManager(IPoolManager(manager));
+
+        (currency0, currency1) = deployMintAndApprove2Currencies();
+        (key,) = initPoolAndAddLiquidity(
+            currency0,
+            currency1,
+            IHooks(address(skipCallsTestHook)),
+            FeeLibrary.DYNAMIC_FEE_FLAG,
```

Comment:
Why is it also a dynamic fee hook? Seems unnecessary for the purpose of this test?